.github/workflows/gh-actions.yml

```diff
@@ -150,3 +150,91 @@ jobs:
         run: sudo apt-get install gcc-multilib
       - name: UBSAN - jerry-tests
         run: $RUNNER --jerry-tests $SKIP_LIST $BUILD_OPTIONS -q
+
+  Linux_ARMv7l_Tests:
+    runs-on: ubuntu-20.04
+    env:
+      RUNTIME: qemu-arm-static
+      TIMEOUT: 300
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          sudo apt update
+          sudo apt install gcc-arm-linux-gnueabihf libc6-dev-armhf-cross qemu-user-static
```

Comment:
Is the yaml block syntax needed? Could this be
```yaml
- run: sudo apt update
- run: sudo apt install gcc-arm-linux-gnueabihf libc6-dev-armhf-cross qemu-user-static
```
?
(It's shorter and the two commands get green/red status separately, which may be helpful in case of an error.)